clang-tools-extra/test/clang-tidy/checkers/bugprone/implicit-widening-of-multiplication-result-constants.cpp

```diff
@@ -0,0 +1,81 @@
+// RUN: %check_clang_tidy -check-suffixes=ALL,NI %s bugprone-implicit-widening-of-multiplication-result %t -- -- -target x86_64-unknown-unknown -x c++
```

Comment:
i have feeling that this test isn't needed here at all, and second would be sufficient, basic scenarios could be covered in implicit-widening-of-multiplication-result.cpp file.